scipy/interpolate/tests/test_bsplines.py

```diff
@@ -1525,6 +1525,26 @@ def test_shapes(self):
         b = make_interp_spline(x, y, k, bc_type=(d_l, d_r))
         assert b.c.shape == (n + k - 1, 5, 6, 7)
 
+    @pytest.mark.parametrize("axis", range(1, 4))
+    def test_shapes_axis(self, axis):
```

Comment:
This is OK for now, but I can imagine a C vs F contiguity bug that gives correct shapes but incorrect numbers. In the course of reviewing new `axis` arguments, I'd like to write a common test `@parametrize`d over the various spline functions to confirm that values match those you'd get by looping over 1D slices.